### PR TITLE
Basic voting "hide abstain" option

### LINKED ISSUE #1120: Add space param to Hide "Abstain" result on Basic voting and calculate % on results based only on "For" and "Against" choices

### PR DESCRIPTION
**Pull request by mktcode**
Fixes #1120

Hub PR: https://github.com/snapshot-labs/snapshot-hub/pull/203

image

`hideAbstain = false`
image

`hideAbstain = true`
image

Using multiple strategies (2x `eth-balance`) to see the split progress bar.

### CODE REVIEW

File: src/components/Block/Results.vue [hunk 1]
```diff
@@ -33,45 +37,64 @@ const choices = computed(() =>
...
     <div v-for="choice in choices" :key="choice.i">
-      <div class="link-color mb-1">
-        <span
-          v-tippy="{
-            content: choice.choice.length > 12 ? choice.choice : null
-          }"
-          class="mr-1"
-          v-text="_shorten(choice.choice, 'choice')"
-        />
+      <template v-if="!(proposal.type === 'basic' && hideAbstain && choice.i === 2)">
```
Comment (mktcode) [inline src/components/Block/Results.vue:40]: changes here summarized:
- template tag wrapper to hide abstain option
- additional versions (for, against) of the last span element (the percentage)
- different max value for progress bar

**Comment by ChaituVR** [comment]
How to set 'hide abstain' in space settings?

**Comment by mktcode** [comment]
> How to set 'hide abstain' in space settings?

Currently only in db. Shall I add a checkbox or something on the settings page @bonustrack?

**Comment by bonustrack** [comment]
@mktcode Yes we should have a checkbox in the settings page, can you add it?

**Comment by samuveth** [comment]
This is ready for a review? Seems the option in settings is still missing

**Comment by mktcode** [comment]
Added the option now:
image

Requires snapshot.js update: https://github.com/snapshot-labs/snapshot.js/pull/441
(here and in the hub)

File: src/locales/default.json [hunk 2]
```diff
@@ -221,7 +221,8 @@
     "voting": "Voting",
     "quorum": "Quorum",
     "type": "Type",
-    "anyType": "Any"
+    "anyType": "Any",
+    "hideAbstain": "Ignore abstain votes in basic voting results."
```
Comment (samuveth) [inline src/locales/default.json:225]: What does "in basic voting results" mean?

Comment (samuveth) [reply src/locales/default.json:225]:
```suggestion
    "hideAbstain": "Ignore abstain votes in basic voting results"
```

File: src/components/Block/Results.vue (hunk 1 above)
Comment (samuveth) [reply src/components/Block/Results.vue:40]: Cool, I didn't know about using `template` to conditional multiple tags without using a parent tag in vue.

File: src/locales/default.json (hunk 2 above)
Comment (samuveth) [inline src/locales/default.json:225]:
```suggestion
    "hideAbstain": "Ignore abstain votes in basic voting results"
```

Comment (mktcode) [reply src/locales/default.json:225]: I don't get it. What's wrong with the dot in the end? It's an imperative sentence. 😃

Comment (bonustrack) [reply src/locales/default.json:225]: It's the title of the field, not a sentence, so no dot

**Comment by mktcode** [comment]
Snapshot.js is updated. Anything else left?

**Comment by bonustrack** [comment]
@mktcode We will merge it soon, Bancor team is still testing it

**Pull request merged**